definitions/GarrGivenName.dbd

```diff
@@ -1,9 +1,9 @@
 COLUMNS
-uint field_2710791686?
 string field_978907121?
 uint field_3259647487?
+int ID
 
 BUILD 6.0.1.18179
-field_2710791686<32>
+$id$ID<32>
```

Comment:
And here. I’d either have them all as guesses or none. Your and martin’s vote which direction.